drivers/drivers.go

```diff
@@ -20,8 +20,9 @@ import (
 )
 
 var ext2mime = map[string]string{
-	".ts":  "video/mp2t",
-	".mp4": "video/mp4",
+	".ts":   "video/mp2t",
+	".mp4":  "video/mp4",
+	".m3u8": "application/x-mpegURL",
```

Comment:
WDYT of going full lower-case here, like what's returned by the `mime` lib when it does work?

Reply:
```suggestion
	".m3u8": "application/x-mpegurl",
```

Reply:
Sounds good, I was just going off https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/StreamingMediaGuide/DeployingHTTPLiveStreaming/DeployingHTTPLiveStreaming.html but shouldn't make a difference either way